package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@date-io/date-fns": "2.x",
+    "@date-io/date-fns": "1.x",
```

Comment:
Btw, if this wasn't the issue I suggest reverting it back to the newer version.

Reply:
If it is the issue, then I would file a separate issue to investigate for later

Reply:
It seems to have fixed it. I made a separate issue for later use.